qutebrowser/config/configfiles.py

```diff
@@ -93,9 +94,25 @@ def init_save_manager(self, save_manager):
         """
         save_manager.add_saveable('yaml-config', self._save)
 
+    def __getitem__(self, name):
+        return self._values[name]
```

Comment:
This isn't covered - it's probably best to just adjust all tests which access `_values` to use this instead anyways?